### PR TITLE
[it] Localizzazione termine Event-Driven Architecture

### PR DESCRIPTION
**Pull request by ugho16**
Signed-off-by: Paolo <ugho16@users.noreply.github.com>

### Describe your changes
Added Event-Driven architecture Italian localization.

### Checklist before opening this PR (put `x` in the checkboxes)
- [ x ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [ x ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/it/event-driven-architecture.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Architettura ad eventi
+status: Completed
+category: concetto
+---
+
+## Cos'è
+
+Un'architettura ad eventi è un'architettura software che promuove la creazione, la processazione e la consumazione di eventi.
```
Comment (annalisag-spark) [inline content/it/event-driven-architecture.md:9]:
```suggestion
Un'architettura basati sugli eventi è un paradigma di progettazione software che promuove la creazione, l'elaborazione e il consumo di eventi.
```

File: content/it/event-driven-architecture.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Quali problematiche affronta
+
+Con l'aumento dei dati in tempo reale, diventa sempre più difficile trovare modi affidabili per assicurarsi che gli eventi siano acquisiti e indirizzati al corretto [servizio](/it/service/) che deve processare la richiesta dell'evento.
+I metodi tradizionali di gestione degli eventi spesso non hanno modo di garantire che i messaggi siano correttamente indirizzati o che siano effettivamente inviati o ricevuti.
+Nel momento in cui le applicazioni iniziano ad essere scalate, diventa più difficile orchestrare gli eventi.
+
+## In che modo aiuta
+
+Le architetture ad eventi costituiscono un fulcro centrale per tutti gli eventi (ad esempio Kafka).
```
Comment (annalisag-spark) [inline content/it/event-driven-architecture.md:22]:
```suggestion
Le architetture ad eventi ne costituiscono un fulcro centrale (ad esempio Kafka).
```

File: content/it/event-driven-architecture.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Si possono definire i produttori degli eventi (origine) e i consumatori (destinatari), e il fulcro centrale garantisce il flusso degli eventi.
```
Comment (annalisag-spark) [inline content/it/event-driven-architecture.md:23]:
```suggestion
All'interno di questo tipo di architettura, infatti, si possono definire i produttori degli eventi (origine) e i loro consumatori (destinatari), e il fulcro centrale ne garantisce il flusso corretto.
```

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

File: content/it/event-driven-architecture.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Architettura ad eventi
+status: Completed
+category: concetto
+---
+
+## Cos'è
+
+Un'architettura basati sugli eventi è un paradigma di progettazione software che promuove la creazione, l'elaborazione e il consumo di eventi.
```
Comment (ugho16) [inline content/it/event-driven-architecture.md:9]: ``
```suggestion
Un'architettura basata sugli eventi è un paradigma di progettazione software che promuove la creazione, l'elaborazione e il consumo di eventi.
```

**Review by annalisag-spark — APPROVED** [review]

**Review by fsbaraglia — APPROVED** [review]

**Pull request merged**